who-we-are/people/davidchristiansen.markdown

```diff
@@ -0,0 +1,11 @@
+---
+name: David Thrane Christiansen
+email: david@haskell.foundation
+title: Executive Director
+executiveTeam: "True"
+tenureStart: 2022-05-02
+image: /assets/images/exec-team/dtc.png
+---
```

Comment:
You mentioned to me elsewhere that you might want a link to your website nearby. You can add one with a `website: https://blah/bah` metadata item here.

Reply:
Ah, I figured it out - it needed a quick tweak of the template as well. Thanks!